proto/stat.proto

```diff
@@ -48,20 +48,18 @@ message PointStat {
   double value = 2;
   // Full metadata of the stat
   StatMetadata metadata = 3;
-  // A hash string of the meta data, this is to be used together with a map from
+  // Hash of the metadata, this is to be used together with a map from
   // the hash to the full metatdata.
-  string meta_hash = 4;
+  uint32 meta_hash = 4;
 }
 
 message PlacePointStat {
   // Keyed by place DCID.
   map<string, PointStat> stat = 1;
-  // Keyed by metadata hash.
-  map<string, StatMetadata> metadata = 2;
 }
 
 message PlacePointStatAll {
-  // Keyed by place DCID.
+  // A list of PlacePointStat, with each one from one source.
```

Comment:
Do you mean each PlacePointStat corresponds to one series?  Then, can we not have meta_hash in PlacePointStat instead of PointStat ?

Reply:
Or perhaps you need it there for GetStatSetResponse ?

Reply:
Right it's needed for GetStatSetResponse. I added meta_hash field for PlacePointStat as well, so PlacePointStatAll only has one meta_hash for each source now.